### PR TITLE
raidboss: Tower of Babil initial timeline

### PR DESCRIPTION
**Pull request by JLGarber**
Untested in game, successfully survived More Than One pass through `test_timeline`.

### CODE REVIEW

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+# BARNABAS #
+#~~~~~~~~~~#
+
+# -ii 6076 61DE
+
+0.0 "--sync--" sync / 00:0839::Magitek Servicing will be sealed off/ window 0,1
+8.9 "Ground and Pound" sync / 1[56]:[^:]*:Barnabas:6247:/ window 8.9,5
+27.3 "Ground and Pound" sync / 1[56]:[^:]*:Barnabas:62EA:/
+
+40.5 "Dynamic Scrapline" sync / 1[56]:[^:]*:Barnabas:62F0:/ window 40.5,10
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:19]: Is there only one id for the negative/positive here?

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+1346.3 "Mighty Blow"
+1354.9 "Mighty Blow"
+1362.4 "Thermal Suppression"
+1368.0 "Magitek Ray"
+
+#~~~~~~~#
+# ANIMA #
+#~~~~~~~#
+
+# -ii 52BE 5E68 6305
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:96]: Can you add targetable lines here?

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+2149.1 "Coffin Scratch x5" sync / 1[56]:[^:]*:Anima:630E:/ duration 5
+2155.7 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/
+2164.2 "Obliviating Claw #1" sync / 1[56]:[^:]*:Anima:630B:/
+2171.0 "Obliviating Claw #2" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2176.0 "Obliviating Claw #" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2182.0 "Charnel Claw #1" sync / 1[56]:[^:]*:Iron Nail:630D:/
+2187.0 "Charnel Claw #2?" sync / 1[56]:[^:]*:Iron Nail:630D:/ # can be skipped?
+2189.5 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/ window 20,20
+2199.1 "Oblivion" sync / 1[56]:[^:]*:Anima:630F:/ duration 5
+2205.8 "--sync--" sync / 1[56]:[^:]*:Anima:5D40:/ window 2205.8,30
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:126]: Is this an hp% based phase? If it is, can you add a TODO that there may be more mechanics?

Can you also add a brief comment or whitespace here to separate out the different phases?

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt (hunk 1 above)
Comment (JLGarber) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:19]: No, good catch. I skipped over that when generating the initial log and it didn't show up again.

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+2121.6 "Obliviating Claw #3" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2127.6 "Charnel Claw #1" sync / 1[56]:[^:]*:Iron Nail:630D:/
+2132.6 "Charnel Claw #2" sync / 1[56]:[^:]*:Iron Nail:630D:/
+2135.1 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/
+2143.6 "Obliviating Claw" sync / 1[56]:[^:]*:Anima:630A:/
+2149.1 "Coffin Scratch x5" sync / 1[56]:[^:]*:Anima:630E:/ duration 5
+2155.7 "--sync--" sync / 1[56]:[^:]*:Anima:6A5C:/
+2164.2 "Obliviating Claw #1" sync / 1[56]:[^:]*:Anima:630B:/
+2171.0 "Obliviating Claw #2" sync / 1[56]:[^:]*:Iron Nail:630C:/
+2176.0 "Obliviating Claw #" sync / 1[56]:[^:]*:Iron Nail:630C:/
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:121]: Is this meant to be #3?

Comment (JLGarber) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:121]: Indeed it is

File: ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt [hunk 5]
```diff
@@ -0,0 +1,158 @@
...
+
+40.5 "Dynamic Scrapline" sync / 1[56]:[^:]*:Barnabas:(6246|62F0):/ window 40.5,10
+41.6 "--sync--" sync / 1[56]:[^:]*:Barnabas:61DD:/
+43.0 "Electromagnetic Release" sync / 1[56]:[^:]*:Barnabas:62F1:/
+55.2 "Shocking Force" sync / 1[56]:[^:]*:Barnabas:62EC:/
+72.2 "Dynamic Pound" sync / 1[56]:[^:]*:Barnabas:(6245|62EE):/ window 72.2,10
+73.3 "--sync--" sync / 1[56]:[^:]*:Barnabas:6075:/
+74.7 "Electromagnetic Release" sync / 1[56]:[^:]*:Barnabas:62EF:/
+85.3 "Thundercall" sync / 1[56]:[^:]*:Barnabas:62ED:/ window 85.3,10
+91.8 "Rolling Scrapline" sync / 1[56]:[^:]*:Barnabas:62EB:/
```
Comment (quisquous) [inline ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:27]: Does Rolling Scrapline have two ids as well?

Comment (JLGarber) [reply ui/raidboss/data/06-ew/dungeon/the_tower_of_babil.txt:27]: Not that I saw across both our logs, and there's only one entry on [XIVAPI](https://xivapi.com/search?indexes=action&string=rolling+scrapline&columns=Name,Name_de,Name_fr,Name_ja,ID).

**Pull request merged**